ui/webui/src/components/storage/InstallationDestination.jsx

```diff
@@ -158,7 +158,7 @@ const LocalStandardDisks = ({ idPrefix, onAddErrorNotification }) => {
                   id="installation-destination-table-label"
                 >
                     {cockpit.format(
-                        cockpit.ngettext("$0 (of $1) disk selected", "$0 (of $1) disks selected", selectedDisksCnt),
+                        "$0 (of $1) disks selected",
```

Comment:
This is incorrect in most languages. See https://www.gnu.org/software/gettext/manual/html_node/Plural-forms.html after "The following rules are known at this point.".

Reply:
FWIW, this is a design problem, not a PR problem, really.